File: program/convert-days-to-hours/convertDaysToHours.scala
def convertDaysToHours(days: Int): Int = {
  var total_hours: Int = days * 24 ;
  return total_hours ;
};

var days_to_convert: Int = 5 ;
var hours:Int =  convertDaysToHours(days_to_convert) ; 
println(hours) ;

File: program/convert-days-to-hours/ConvertDaysToHours.c
#include <stdio.h> /* printf, scanf*/

int ConvertDaysToHours(int days);

int main()
{
    int num = 0;

    int is_good = scanf("%d",&num);

    if(is_good != 1)
    {
        return -1;
    }

    printf("%d\n",ConvertDaysToHours(num));

    return 0;
}

int ConvertDaysToHours(int days)
{
    return days * 24;
}